packages/next/src/server/lib/trace/tracer.ts

```diff
@@ -229,6 +229,18 @@ class NextTracerImpl implements NextTracer {
             options: { ...fnOrOptions },
           }
 
+    const spanName = options.spanName ?? type
+
+    if (process.env.NEXT_OTEL_LOG_PREFIX && LogSpanAllowList.includes(type)) {
+      performance.measure(
+        `${process.env.NEXT_OTEL_LOG_PREFIX}:next-${spanName.replace(
```

Comment:
The `spanName` seems wrong here. E.g. it's normally something "find page components", so this regex doesn't do anything. It's better to use `type`, minus, which is "NextNodeServer.findPageComponents". Might be good to remove "NextNodeServer".

Reply:
Good point, updated